.github/workflows/prerelease.yml

```diff
@@ -2,6 +2,8 @@ name: Prerelease pipeline
 
 on:
   release:
+    branches:
```

Comment:
🤔

Reply:
Please ignore .. I was trying to setup the go-lint to run for my branch. Its been reverted after test